src/server.js

```diff
@@ -168,8 +181,16 @@ function run(db) {
         db.run('INSERT OR IGNORE INTO user (id,name,blocked,trusted,created_at) VALUES (1,"dev",0,1,datetime())');
     }
 
-    var server = app.listen(config.get('port'), config.get('host'), (err) => {
-        if (err) throw err;
-        console.log(`server listening on ${server.address().port}`);
-    });
+    if (configSsl && configSsl.certificate_path) {
+        server = https.createServer( sslOptions, app );
```

Comment:
Something doesn't seem quite right with the server var being created.

Either we need to use a let above the if statement `let server;`

And then remove the `var` on the server below.

Or we need to make this `const server` on this line, and then change the `var server` below to be `const server` also.

Reply:
I've changed this to `let`; as the `if` statement is required; If the next Dev removes the certificate, or if this is deployed across an insecure connection, the server needs the ability to serve over `http` - which uses a completely different library to the secured connection over `https`.